src/internal/group_api/requests.rs

```diff
@@ -484,24 +490,22 @@ pub mod group_remove_entity {
                     users: removed_users,
                 },
                 error_code,
-                AuthV2Builder::new(auth, Utc::now()),
+                AuthV2Builder::new(auth, OffsetDateTime::now_utc()),
             )
             .await
     }
 }
 
 #[cfg(test)]
 mod tests {
-    use chrono::TimeZone;
-
     use super::*;
     use recrypt::api::KeyGenOps;
 
     ///This test is to ensure our lowercase admin and member permissions are handled correctly.
     #[test]
     fn group_item_serde_format_is_expected() {
-        let created = Utc.timestamp_millis(1_551_461_529_000);
-        let updated = Utc.timestamp_millis(1_551_461_529_001);
+        let created = OffsetDateTime::from_unix_timestamp_nanos(1_551_461_529_000_000).unwrap();
```

Comment:
This isn't right. This is microseconds. Need to investigate.

Reply:
The test didn't really care about this number... I did increase it by another factor of 1000.